flashcard_model.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+
+class FlashcardModel
+  attr_reader :file, :answers, :parsed_file
+
+  def initialize(file)
+    @file = file
+    @answers = []
+    @parsed_file = []
+    create_nested_array
+  end
+
+  def array_creator
+    txt = File.open(file, "r+")
+    file_array = txt.map do |row|
+      if row == "\n"
+        row = false
+      end
+        row
+    end
+    file_array
+  end
+
+  def array_chomper
+    file_array = array_creator
+    clean_file_array = file_array.map do |row|
+      if row.class == String
+        row = row.chomp
+      end
+      row
+    end
+    clean_file_array
+  end
+
+  def create_nested_array
+    file_array = array_chomper
+    while file_array.length > 0
+      parsed_file << file_array.slice!(0..2)
+    end
+  end
```

Comment:
Try to avoid using data types in variables and method names. It's more descriptive and helpful for developers looking at your code for the first time if you name things based off of their purpose to your code instead of their makeup.